src/chapters/http/http-in-the-browser.rst

```diff
@@ -0,0 +1,62 @@
+HTTP in the Browser
+===================
```

Comment:
Wondering if they all are using Firefox?  I have Chrome and Safari on my computer and it looks a bit different, but I did find the network tab in my developers tools, which looked similar.  I don't see if its a get or post request.  Maybe a note about different browser developer tools being similar but different and how to locate the developer tools?